### PR TITLE
Update debug-live-azure-apps-troubleshooting.md

### PR DESCRIPTION
**Pull request by WilliamXieMSFT**
Add troubleshooting steps for clearing out the MRU (Most Recently Used) lists.

### CODE REVIEW

**Comment by PRMerger3** [comment]
@WilliamXieMSFT : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/debugger/debug-live-azure-apps-troubleshooting.md [hunk 1]
```diff
@@ -191,6 +191,20 @@ To fix this, delete the following App settings in the Azure portal and attach th
...
 
+## Issue: I am attaching to the wrong/old Azure resource or storage account
+
+Take these steps:
+
+The "Azure resource" and "Storage account" entries use resource names as keys so actions such as migrating a resource to different subscriptions can cause problems.
+To clear out the list, follow these steps:
+
+1.	Run these commands in Developer Command Prompt for VS (with admin privileges)
+```
```
Comment (Mikejo5000) [inline docs/debugger/debug-live-azure-apps-troubleshooting.md:202]:
```suggestion
   ```
```suggestion
```
```
```

Comment (Mikejo5000) [inline docs/debugger/debug-live-azure-apps-troubleshooting.md:202]:
```suggestion
   ```
```suggestion
```
```
```

File: docs/debugger/debug-live-azure-apps-troubleshooting.md [hunk 2]
```diff
@@ -191,6 +191,20 @@ To fix this, delete the following App settings in the Azure portal and attach th
...
+vsregedit remove local HKCU SnapshotDebugger AzureResourcesMRU
```
Comment (Mikejo5000) [inline docs/debugger/debug-live-azure-apps-troubleshooting.md:203]:
```suggestion
   vsregedit remove local HKCU SnapshotDebugger AzureResourcesMRU
```
```suggestion
vsregedit remove local HKCU SnapshotDebugger AzureResourcesMRU
```

Comment (Mikejo5000) [inline docs/debugger/debug-live-azure-apps-troubleshooting.md:203]:
```suggestion
   vsregedit remove local HKCU SnapshotDebugger AzureResourcesMRU
```
```suggestion
vsregedit remove local HKCU SnapshotDebugger AzureResourcesMRU
```

File: docs/debugger/debug-live-azure-apps-troubleshooting.md [hunk 3]
```diff
@@ -191,6 +191,20 @@ To fix this, delete the following App settings in the Azure portal and attach th
...
+vsregedit remove local HKCU SnapshotDebugger StorageAccountsMRU
```
Comment (Mikejo5000) [inline docs/debugger/debug-live-azure-apps-troubleshooting.md:204]:
```suggestion
   vsregedit remove local HKCU SnapshotDebugger StorageAccountsMRU
```
```suggestion
vsregedit remove local HKCU SnapshotDebugger StorageAccountsMRU
```

Comment (Mikejo5000) [inline docs/debugger/debug-live-azure-apps-troubleshooting.md:204]:
```suggestion
   vsregedit remove local HKCU SnapshotDebugger StorageAccountsMRU
```
```suggestion
vsregedit remove local HKCU SnapshotDebugger StorageAccountsMRU
```

File: docs/debugger/debug-live-azure-apps-troubleshooting.md [hunk 4]
```diff
@@ -191,6 +191,20 @@ To fix this, delete the following App settings in the Azure portal and attach th
...
+```
```
Comment (Mikejo5000) [inline docs/debugger/debug-live-azure-apps-troubleshooting.md:205]:
```suggestion
   ```
```suggestion
```
```
```

Comment (Mikejo5000) [inline docs/debugger/debug-live-azure-apps-troubleshooting.md:205]:
```suggestion
   ```
```suggestion
```
```
```

**Comment by PRMerger19** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger10** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/debugger/debug-live-azure-apps-troubleshooting.md [hunk 5]
```diff
@@ -191,6 +191,18 @@ To fix this, delete the following App settings in the Azure portal and attach th
...
 
+## Issue: I am attaching to the wrong/old Azure resource or storage account
+
+Take these steps:
+
+The "Azure resource" and "Storage account" entries use resource names as keys so actions such as migrating a resource to different subscriptions can cause problems.
+To clear out the list, follow these steps:
+
+1.	Run these commands in Developer Command Prompt for VS (with admin privileges)
+   vsregedit remove local HKCU SnapshotDebugger AzureResourcesMRU
```
Comment (Mikejo5000) [inline docs/debugger/debug-live-azure-apps-troubleshooting.md:202]:
```suggestion
   ```
   vsregedit remove local HKCU SnapshotDebugger AzureResourcesMRU
```

**Comment by PRMerger17** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger18** [comment]
@Mikejo5000 : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Mikejo5000 — APPROVED** [review]

**Comment by Mikejo5000** [comment]
#sign-off Thanks @WilliamXieMSFT !

**Pull request merged**